jablib/src/main/java/org/jabref/logic/ai/util/ChatMessageUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.jabref.logic.ai.util;
+
+import java.util.Optional;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.UserMessage;
+
+public class ChatMessageUtils {
+
+    private ChatMessageUtils() {
+    }
+
+    public static Optional<String> getContent(ChatMessage chatMessage) {
+        if (chatMessage == null) {
+            return Optional.empty();
+        }
+
+        String content = switch (chatMessage) {
+            case UserMessage user ->
+                    user.singleText();
+            case AiMessage ai ->
+                    ai.text();
+            case ErrorMessage err ->
+                    err.getText();
+            default ->
+                    null;
+        };
+
+        return Optional.ofNullable(content);
+    }
+}
```

Comment:
I think it is unneccessary to create a new utility class for just that simple method. YAGNI!
Put it into ChatMassageComponent

Reply:
Yeah, could be, maybe. Eventually...
Until then: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it